cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -278,7 +278,7 @@ body {
             border: none;
             border-right: 1px solid @gray-40;
             color: @black;
-            font-size: unit( 18px / @base-font-size-px, em );
+            font-size: 18px;
```

Comment:
Now that this is not an icon font, we may want to change how we size the hamburger icon, but this works fine for now.